src/boilerplate/sw.h

```diff
@@ -36,6 +36,11 @@
  */
 #define SW_WRONG_DATA_LENGTH 0x6A87
 
+/**
+ * Status word for fail in Swap
+ */
+#define SW_FAIL_SWAP 0x6A8E
+
```

Comment:
Is this status word also used in other apps for the swap integration?
I would have preferred not to use an SW in the `0x6AXX` range, and maybe use `0x6B00` or something similar. Could also use a subrange like `0x6BXX` for swap, so that different error conditions could be detected appropriately.

Reply:
It is used by some other apps, but I can change it for sure :) 
I'll go for `0x6B00`